sound/soc/sof/trace.c

```diff
@@ -43,6 +43,12 @@ static size_t sof_wait_trace_avail(struct snd_sof_dev *sdev,
 	if (ret)
 		return ret;
 
+	/* tracing has ended, signal EOF to client */
+	if (!sdev->dtrace_is_enabled && sdev->dtrace_eof) {
+		sdev->dtrace_eof = 0;
```

Comment:
I am missing what this new dtrace_eof actually does. It looks like a state value but I don't see any action taken nor any 'notification to client'. What am I missing?

Reply:
@plbossart See above, dtrace_eof is internal booking, a request to signal eof, the actual EOF signal is returning 0 from read().